### PR TITLE
Mobile movie tile @ media for small 

### PR DESCRIPTION
**Pull request by DorotaZielinska**
I have gave name of Mobile, but in theme we have that as small, because I m not sure if we need the mobile size witch is 320px?

### CODE REVIEW

File: src/common/TileList/styled.js [hunk 1]
```diff
@@ -9,30 +9,57 @@ export const MovieTileList = styled.article`
     box-shadow: 0px 4px 12px 0px rgba(186, 199, 213, 0.50);
     display: grid;
     grid-auto-rows: auto 1fr;
+
+    @media (max-width: ${({ theme }) => theme.breakpoint.small}) {
+        grid-template-columns: auto 1fr;
+        grid-gap: 16px;
+        width: 320px;
```
Comment (MirekSobota) [inline src/common/TileList/styled.js:16]: If you put max-width: 320px; here, you can create padding from the left and right sides, similar to Figma.

File: src/features/MovieList/styled.js [hunk 2]
```diff
@@ -12,6 +16,10 @@ export const MoviesList = styled.ul`
     display: grid;
     grid-template-columns: repeat(4, 1fr);
     grid-gap: 24px;
+    
+    @media (max-width: ${({ theme }) => theme.breakpoint.small}) {
+        grid-template-columns: 1fr;
+    }
```
Comment (MirekSobota) [inline src/features/MovieList/styled.js:22]: You can add here ` padding-right: 16px; `

**Review by MirekSobota — COMMENTED** [review]
With these styles, I would use @media (max-width: 320px) because if you make the screen bigger, we get a bit of a gap on the right side.

File: src/common/TileList/styled.js (hunk 1 above)
Comment (DorotaZielinska) [reply src/common/TileList/styled.js]: Thank I added change.

File: src/features/MovieList/styled.js (hunk 2 above)
Comment (DorotaZielinska) [reply src/features/MovieList/styled.js:23]: added change thanks :D

**Review by MirekSobota — APPROVED** [review]

File: src/features/MovieList/styled.js [hunk 3]
```diff
@@ -3,7 +3,11 @@ import styled from "styled-components";
 export const MovieListTitle = styled.h1`
     font-size: 36px;
```
Comment (PawelOwiesek) [inline src/features/MovieList/styled.js:4]: I think You are missing line-height 1.2

File: src/common/TileList/styled.js [hunk 4]
```diff
@@ -9,30 +9,58 @@ export const MovieTileList = styled.article`
...
 `;
 
 export const Content = styled.div`
     height: 100%;
     display: flex;
     flex-direction: column;
     justify-content: space-between;
+
+    @media (max-width: ${({ theme }) => theme.breakpoint.small}) {
+       justify-content: flex-start;
```
Comment (PawelOwiesek) [inline src/common/TileList/styled.js:40]: great solution👌

File: src/common/TileList/styled.js [hunk 5]
```diff
@@ -9,30 +9,58 @@ export const MovieTileList = styled.article`
...
 
 export const StyledTitle = styled.h1`
     font-weight: 500;
     font-size : 22px;
     color: ${({ theme }) => theme.color.woodsmoke};
     margin-bottom: 8px;
+
+    @media (max-width: ${({ theme }) => theme.breakpoint.small}) {
+       font-size: 16px;
+       margin-bottom: 4px;
```
Comment (PawelOwiesek) [inline src/common/TileList/styled.js:52]: we can write this like :margin:0 0 4px;

File: src/common/TileList/styled.js [hunk 6]
```diff
@@ -9,30 +9,58 @@ export const MovieTileList = styled.article`
...
     display: flex;
     flex-direction: column;
     justify-content: space-between;
+
+    @media (max-width: ${({ theme }) => theme.breakpoint.small}) {
+       justify-content: flex-start;
+    };
 `;
 
 export const StyledTitle = styled.h1`
```
Comment (PawelOwiesek) [inline src/common/TileList/styled.js:44]: think missing line-height = 1.3

File: src/common/Rating/styled.js [hunk 7]
```diff
@@ -7,17 +7,39 @@ export const RatingContainer = styled.div`
...
+export const IconWrapper = styled.div`
+        svg{
+            @media (max-width: ${({ theme }) => theme.breakpoint.small}) {
+                width: 16px;
+                height: 16px;
+            };
+        }; 
 `;
 
 export const Vote = styled.span`
```
Comment (PawelOwiesek) [inline src/common/Rating/styled.js:26]: line-height=1.3

File: src/common/Rating/styled.js [hunk 8]
```diff
@@ -7,17 +7,39 @@ export const RatingContainer = styled.div`
...
     font-size: 16px;
     font-weight: 600;
     color:${({ theme }) => theme.color.woodsmoke};
+
+    @media (max-width: ${({ theme }) => theme.breakpoint.small}) {
+       font-size: 13px;
+    };
 `;
 
 export const Votes = styled.span`
```
Comment (PawelOwiesek) [inline src/common/Rating/styled.js:36]: line-height=1.5

**Review by PawelOwiesek — COMMENTED** [review]
Minor changes like line-height and gap in one place are needed, other than that everything looks great

File: src/common/Rating/styled.js [hunk 9]
```diff
@@ -7,17 +7,39 @@ export const RatingContainer = styled.div`
...
+            };
+        }; 
 `;
 
 export const Vote = styled.span`
     font-size: 16px;
     font-weight: 600;
     color:${({ theme }) => theme.color.woodsmoke};
+
+    @media (max-width: ${({ theme }) => theme.breakpoint.small}) {
```
[the remark this answers is not in the document]
Comment (DorotaZielinska) [reply src/common/Rating/styled.js:32]: added thankd :D

File: src/common/Rating/styled.js [hunk 10]
```diff
@@ -7,17 +7,39 @@ export const RatingContainer = styled.div`
...
+    };
 `;
 
 export const Votes = styled.span`
     width: auto;
     font-size: 16px;
     font-weight: 400;
     color: ${({ theme }) => theme.color.waterloo};
+
+    @media (max-width: ${({ theme }) => theme.breakpoint.small}) {
```
[the remark this answers is not in the document]
Comment (DorotaZielinska) [reply src/common/Rating/styled.js:45]: added thankd :D

File: src/common/Rating/styled.js (hunk 8 above)
Comment (DorotaZielinska) [reply src/common/Rating/styled.js:38]: added thankd :D

File: src/common/Rating/styled.js (hunk 7 above)
Comment (DorotaZielinska) [reply src/common/Rating/styled.js:26]: added thankd :D

File: src/common/TileList/styled.js [hunk 11]
```diff
@@ -9,30 +9,58 @@ export const MovieTileList = styled.article`
...
     margin-bottom: 8px;
+
+    @media (max-width: ${({ theme }) => theme.breakpoint.small}) {
+       font-size: 16px;
+       margin-bottom: 4px;
+       margin-top: 0px;
+    };
 `;
 
 export const StyledSubTitle = styled.h2`
```
[the remark this answers is not in the document]
Comment (DorotaZielinska) [reply src/common/TileList/styled.js:57]: added thankd :D

File: src/common/TileList/styled.js (hunk 6 above)
Comment (DorotaZielinska) [reply src/common/TileList/styled.js:44]: added thankd :D

File: src/common/TileList/styled.js (hunk 5 above)
Comment (DorotaZielinska) [reply src/common/TileList/styled.js]: added thankd :D

File: src/features/MovieList/styled.js (hunk 3 above)
Comment (DorotaZielinska) [reply src/features/MovieList/styled.js:5]: added thankd :D

**Review by PawelOwiesek — APPROVED** [review]
Great job, it looks like everything is as designed.

File: src/features/MovieList/styled.js (hunk 2 above)
Comment (DorotaZielinska) [reply src/features/MovieList/styled.js:28]: added thankd :D

**Comment by DorotaZielinska** [comment]
Thanks for finded bugs :D

**Pull request merged**